### ISSUE TITLE
Configuring bringup parameters

### ISSUE THREAD

**Issue by himanshuak** [TooSchoolForCool/EddieBot-ROS#3]
Hello, 
I am using eddie bot for a personal project. I was able to get the robot moving using your repo but I need a little help to configure the set of parameters in `/eddiebot_bringup/launch/includes/eddie_driver.launch.xml`. 
````
    <param name="serial_port" value="/dev/ttyUSB0" />
    <param name="angular_scale" value="1.1" />
    <param name="linear_scale" value="1.0" />
    <param name="left_power" value="90" />
    <param name="right_power" value="90" />
    <param name="rotation_power" value="90" />
    <param name="acceleration_power" value="50" />
    <param name="deceleration_power" value="100" />
    <param name="min_power" value="32"/>
    <param name="left_speed" value="100" />
    <param name="right_speed" value="100" />
    <param name="rotation_speed" value="50" />
    <param name="acceleration_speed" value="50" />
````
I noticed that the robot has a max linear velocity of 0.5 m/s. Is it possible to somehow set the max velocity of the robot? And also the slowest the robot can move is at 0.2 m/s. For velocities lower than that the robot doesn't move. Any suggestion would be of great help. Thanks in advance

**Comment by TooSchoolForCool** [comment]
> I noticed that the robot has a max linear velocity of 0.5 m/s. Is it possible to somehow set the max velocity of the robot? 

Yes, it is possible. Please go through the [eddiebot_bringup](https://github.com/TooSchoolForCool/EddieBot-ROS/tree/master/eddiebot/eddiebot_bringup/src) package, and you will figure out how to achieve this. A quick tip is that the more power you give to the motor, the faster the speed. You can set the upper limit of the power in the source code on your own, but it might not exceed the limit of the firmware & hardware of the robot. 

> And also the slowest the robot can move is at 0.2 m/s. For velocities lower than that the robot doesn't move.

Yes, this is quite normal. If the power cannot overcome the stiction (i.e., "friction" when object does not move yet), the robot does not move. You might change the amount of power each step you decrease to reach a speed slower than 0.2 m/s.
